File: MyBreak/DayThree/DayThree/Java/Comperator.java
import java.util.*;

public class Comperator {

  public static void main(String[] args) {
    creatingComperator();
  }

  public static void creatingComperator() {
    Scanner scan = new Scanner(System.in);

    System.out.println("Enter first number: ");
    int firstNumber = scan.nextInt();

    System.out.println("Enter second number: ");
    int secondNumber = scan.nextInt();

    if (firstNumber == secondNumber) {
      System.out.println("0");

    } else if (firstNumber > secondNumber) {
      System.out.println("1");
    } else if (secondNumber > firstNumber) {
      System.out.println("-1");
    }

  }
}

File: MyBreak/DayThree/JavaScript/Comperator.js
const prompt = require('prompt-sync')()

function createComperator(){
let  firstNumber = prompt("Enter first number ")
let secondNumber = prompt("Enter second number ")

if(firstNumber == secondNumber){
  console.log("0")
}
else if(firstNumber > secondNumber){
  console.log("1")
}
else if(secondNumber > firstNumber){
  console.log("-1")
}
}
createComperator()